web/src/app/models/socialStatsHistory.ts

```diff
@@ -0,0 +1,9 @@
+export interface SocialStatsHistory {
+  provider: string;
+  userId: string;
+  followers: number;
+  following: number;
+  repos?: number;
+  tweets?: number;
```

Comment:
could this be combined for all socials? like `activityCount`?

Reply:
Couldn't get you @eddiejaoude . Please elaborate.

Reply:
now there is `repos`, `tweets` and soon to be things like `posts` etc. Can we combine this value into `activityCount`, therefore when it is GitHub it will be repos, when it is twitter it will be tweets, when it is instagram it will be posts, when it is youtube it will be videos?

Reply:
Okay got it @eddiejaoude  Will change